Makefile

```diff
@@ -3,6 +3,29 @@ TESTS = [api]
 # lagoon-remote, and lagoon-test charts. If IMAGE_TAG is not set, it will fall
 # back to the version set in the CI values file, then to the chart default.
 IMAGE_TAG =
+
+# UI_IMAGE_REPO and UI_IMAGE_TAG are an easy way to override the UI image used
+# only works for installations where INSTALL_STABLE_CORE=false
+# UI_IMAGE_REPO = uselagoon/ui
+# UI_IMAGE_TAG = latest
+
+# SSHPORTALAPI_IMAGE_REPO and SSHPORTALAPI_IMAGE_TAG are an easy way to override the ssh portal api image used in the local stack lagoon-core
+# only works for installations where INSTALL_STABLE_CORE=false
+SSHPORTALAPI_IMAGE_REPO = ghcr.io/uselagoon/lagoon-ssh-portal/ssh-portal-api
+SSHPORTALAPI_IMAGE_TAG = pr-497
+
+# SSHTOKEN_IMAGE_REPO and SSHTOKEN_IMAGE_TAG are an easy way to override the ssh token image used in the local stack lagoon-core
+# only works for installations where INSTALL_STABLE_CORE=false
+SSHTOKEN_IMAGE_REPO = ghcr.io/uselagoon/lagoon-ssh-portal/ssh-token
+SSHTOKEN_IMAGE_TAG = pr-497
```

Comment:
these can be nulled now that pr-497 has been merged to main and released.

Reply:
ah yep